### PR TITLE
feat(redesign): Download page

### PR DESCRIPTION
**Pull request by darkweak**
Capture d’écran 2023-10-14 à 01 10 06
Capture d’écran 2023-10-14 à 01 10 15
Capture d’écran 2023-10-14 à 01 10 28

### CODE REVIEW

**Comment by mholt** [comment]
I need to fix that gradient bg 🧐

Thanks for working on this! Will look at this soon

File: new/download.html [hunk 1]
```diff
@@ -1,24 +1,109 @@
...
+						<option value="openbsd-amd64">OpenBSD amd64</option>
+						<option value="openbsd-arm-6">OpenBSD arm 6</option>
+						<option value="openbsd-arm-7">OpenBSD arm 7</option>
+						<option value="openbsd-arm64">OpenBSD arm64</option>
+						<option value="windows-amd64">Windows amd64</option>
+						<option value="windows-arm-6">Windows arm 6</option>
+						<option value="windows-arm-7">Windows arm 7</option>
+						<option value="windows-arm64">Windows arm64</option>
+					</select>
+					<a id="download-link" href="https://localhost/api/download">
```
Comment (mohammed90) [inline new/download.html]: I believe this should say `https://caddyserver.com/...`, right?

**Comment by mholt** [comment]
I'll be ready to revisit this soon! Just about to finish phase 1 which deploys the new marketing pages.

**Comment by mholt** [comment]
Oops. Can we maybe open this PR again on a new branch with the latest master?

**Pull request closed without merging**